### DISCUSSION TITLE
Oracle Date DataType

### DISCUSSION THREAD

**Discussion by imranzaheer612** [IvorySQL/IvorySQL#185] [Q&A] [answered]
Why the year is being stored as an astronomical year in date datatype?

```
postgres=# SELECT date '4714-11-24 BC';
         date         
----------------------
 -4713-11-24 00:00:00
(1 row)
```
I am using Ivorysql-1.5.
The date style is set to ISO, MDY.
Is this the expected output?

The same test case is used in `ora_date.sql` but the expected output was diff there.
Following output is taken from `ora_date.out`
```
-- Check upper and lower limits of date range
SELECT date '4714-11-24 BC';
          date          
------------------------
 4714-11-24 00:00:00 BC
(1 row)
```

**Comment by tanyang-star** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/92354960/218363426-86385419-0354-462b-a30f-86df820c7d2e.png)
。so i think '-4714-11-24 00:00:00' is right.

**Reply by imranzaheer612** [reply]
I noticed you added PR for that https://github.com/IvorySQL/IvorySQL/pull/191. 
Thanks for helping 😃.
